frb_codegen/src/utils.rs

```diff
@@ -71,6 +71,14 @@ pub fn get_symbols_if_no_duplicates(configs: &[crate::Opts]) -> Result<Vec<Strin
     Ok(all_symbols)
 }
 
+pub fn first<T>(vec: &[T]) -> Option<&T> {
```

Comment:
uber-nit: We already have a function for this https://doc.rust-lang.org/std/vec/struct.Vec.html#method.first